libstuff/STCPNode.cpp

```diff
@@ -176,21 +176,12 @@ void STCPNode::postPoll(fd_map& fdm, uint64_t& nextActivity) {
                             _onMESSAGE(peer, message);
                         }
                     }
-                } catch (const char* e) {
+                } catch (const SException& e) {
```

Comment:
There are a few spots like here where this change looks risky... For all of them, are you 100% sure that the only thing that can be thrown are SExceptions? Sometimes it can be really hard to tell...

Reply:
I went through every instance of `catch` and `throw` in Bedrock.  The standard library never throws `const char*` (only subclasses of `std::exception`), so I'm pretty confident this works.